ext/gtest/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if (WIN32)
 		BINARY_DIR "${PROJECT_BINARY_DIR}/gtest"
 
 		INSTALL_DIR "${PROJECT_BINARY_DIR}/lib"
-		INSTALL_COMMAND "${CMAKE_COMMAND}" -E copy ${PROJECT_BINARY_DIR}/gtest/libgtest.a ${PROJECT_BINARY_DIR}/gtest/libgtest_main.a ${PROJECT_BINARY_DIR}/lib
```

Comment:
1. I do this here to avoid messy files in "${PROJECT_BINARY_DIR}/lib"
2. fixed in https://github.com/qinzuoyan/rDSN/commit/300545e20149294e70486e3891de385aaa616d24